src/MembershipsSDKAdapter.js

```diff
@@ -238,13 +268,18 @@ export default class MembershipsSDKAdapter extends MembershipsAdapter {
 
       // Emit on membership updates
       meeting.members.on('members:update', (payload) => {
+        logger.debug(...LOG_ARGS, 'getMeetingMembers()', ['received members: update event', {payload}]);
         if (payload && payload.full) {
           members$.next(getMembers(payload.full));
         }
       });
     }
 
-    return members$;
+    return members$.pipe(
+      tap((members) => {
+        logger.debug(...LOG_ARGS, 'getMeetingMembers()', [`emitting membership for meeting with id "${meetingID}"`, {members}]);
```

Comment:
this should have the resourceId set to `meeting-${meetingId}` and the message can say just `['emitting membership object', members]`